### PR TITLE
Migrate shortcut

### PR DESCRIPTION
**Pull request by Wgil**
# Description

It adds a command to help migrating smart contracts locally with ease.  It wraps the commands: `store`, `init` and `migrate`.

Usage example:

`sh scripts/migrate.sh ./artifacts/financial_snapshot_contract.wasm`
`sh scripts/migrate.sh ./artifacts/account_history_contract.wasm '{"limit": 10, "value_denom": "uusdc", "expiration": {"at_time": "604800000000000"}}'`

### CODE REVIEW

File: scripts/migrate.sh [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+    # Wait for a few seconds before initiating the contract
+    sleep 1
+
+    # Get contract address based on code ID
+    contract_address=$(elysd q wasm contracts $code_id | awk '/^contracts:/{getline; print $2}')
+
+    # Migrate the contract
+    echo "Migrating the contract..."
+    elysd tx wasm migrate $contract_address $code_id '{}' --from cw --keyring-backend test --chain-id elystestnet-1 --gas auto --gas-adjustment=1.3 --fees 100000uelys -b sync -y
```
Comment (cosmic-vagabond) [inline scripts/migrate.sh:46]: @Wgil thanks so much for preparing this script, is there a reason why both init and migrate are called one after the other, init is supposed to be call once for contract initialization stage and then whenever there is a new version that we want the contract to use we use migrate

Comment (Wgil) [reply scripts/migrate.sh:46]: @cosmic-vagabond sorry I was confused thinking that we needed to execute the three commands for local dev but certainly there is no need for that. I just talked to @kenta-elys and he only uses `store` and `init` in his local workflow.

Comment (Wgil) [reply scripts/migrate.sh]: Just updated it and added an option for extra common params like node, etc.

File: scripts/init_contract.sh [hunk 2]
```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+# This command is used to automate the store and init process of
+# smart contracts. Examples:
+
+# Without params: sh scripts/migrate.sh ./artifacts/financial_snapshot_contract.wasm
+# With init params: sh scripts/migrate.sh ./artifacts/account_history_contract.wasm '{"limit": 10, "value_denom": "uusdc", "expiration": {"at_time": "604800000000000"}}'
+# Common extra aguments: sh scripts/migrate.sh ./artifacts/financial_snapshot_contract.wasm {} '--node test.com'
```
Comment (cosmic-vagabond) [inline scripts/init_contract.sh:8]: @Wgil thanks for those changes, you might wanna update the examples here as well as the file name changed, thank you

Comment (Wgil) [reply scripts/init_contract.sh]: Thanks

**Comment by Wgil** [comment]
All conversations resolved

**Review by cosmic-vagabond — APPROVED** [review]

**Pull request merged**